From 9b31c9a5f99e6d576f44ca28167b6e873492e975 Mon Sep 17 00:00:00 2001
From: Tim Yao <tim.yao@amlogic.com>
Date: Wed, 22 Apr 2015 18:19:13 -0700
Subject: [PATCH 5576/5965] PD#105913: h264 decoder: fix debug message print.

When bit stream has SEI CC data, there are too many verbose
message prints. Remove unnecessary debug messages.

Change-Id: I82fde85179db489bb5b87362c212649f82f33880
---
 drivers/amlogic/amports/vh264.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/amports/vh264.c b/drivers/amlogic/amports/vh264.c
index 703076b0d5c8..d088cab6ca2c 100755
--- a/drivers/amlogic/amports/vh264.c
+++ b/drivers/amlogic/amports/vh264.c
@@ -1473,9 +1473,9 @@ static void vh264_isr(void)
             //daddr = (unsigned char *)(sei_data_buffer_remap + ltemp);
             printk("0x%x\n", *daddr);
         }
-        #endif
         printk("pocinfo 0x%x, top poc %d, wp 0x%x, length %d\n",
             READ_VREG(AV_SCRATCH_L), READ_VREG(AV_SCRATCH_M), sei_itu35_wp, sei_itu35_data_length);
+        #endif
         user_data_poc.poc_info = READ_VREG(AV_SCRATCH_L);
         user_data_poc.poc_number = READ_VREG(AV_SCRATCH_M);
         set_userdata_poc(user_data_poc);
-- 
2.19.0

